spec/models/game_spec.rb

```diff
@@ -43,4 +43,9 @@
       expect(game.white_to_move).to eq(true)
     end
   end
+
+  it "should have correct number of pieces" do
+      game = Game.create
```

Comment:
This line is not actually writing to the DB due to the validations on `Game`. Instead, you should just use the `game` variable created by the `let` line at the top (more info at http://www.relishapp.com/rspec/rspec-core/v/3-4/docs/helper-methods/let-and-let). That line is in turn leveraging the factory_girl factory set up in spec/factories/games.rb.